app/upgrades/v2/upgrades.go

```diff
@@ -0,0 +1,24 @@
+package v2
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	recordstypes "github.com/Stride-Labs/stride/x/records/types"
+)
+
+const (
+	UpgradeName = "Upgrade to Resolve Consensus Bug"
+)
+
+// CreateUpgradeHandler creates an SDK upgrade handler for v2
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		vm[recordstypes.ModuleName] = 2
```

Comment:
I don't think we want to increment the records module consensus version here (just remove this line). I believe that's only if there's a state migration. 

And for future reference, when specifying a new version here, you also have to increment the `ConsensusVersion` in the module's `module.go` and add [migration handlers](https://github.com/Stride-Labs/stride/tree/main/app/upgrades#add-migration-handler)